src/Connectors/AbstractConfig.php

```diff
@@ -26,6 +34,6 @@ protected function validate(array $config): void
 
     protected function getSchemaConfig(string $configName, string $topicId): array
     {
-        return config($configName.'.avro_schemas.'.$topicId, []);
+        return config($configName . '.avro_schemas.' . $topicId, []);
```

Comment:
```suggestion
        return config("{$configName}.avro_schemas.{$topicId}", []);
```
Não acha esse formato mais elegante?

Reply:
E esse valor default `[]` não resolve a necessidade do cast em outros pontos?

Reply:
Não, porque o problema é o que está especificado no retorno da função `@return mixed|\Illuminate\Config\Repository`